examples/games/game_menu.rs

```diff
@@ -433,11 +432,11 @@ mod menu {
                         // Display the game name
                         parent.spawn((
                             Text::new("Bevy Game Menu UI"),
-                            TextStyle {
+                            TextFont {
                                 font_size: 67.0,
```

Comment:
Question: generally we only set the size of the font when creating it. Is it possible that we maybe add a shorthand method for this? like `TextFont::from_size(size: f32)`

Reply:
Yeah I agree it's annoying. `FontSmoothing` is so niche, I'd rather it was a separate component as well. I want to keep this PR to just dealing with `color` though as it's the biggest problem.